### ISSUE TITLE
Bar and Line: line issues

### ISSUE THREAD

**Issue by dansotoz** [Kranf99/kibella#78]
the image should be self explanatory
![newplot(3)](https://user-images.githubusercontent.com/13632722/79035885-241efa00-7b88-11ea-8208-ba57c971f27f.png)

**Comment by Kranf99** [comment]
Here is a small video that explain how to reproduce the error:
[line_error.zip](https://github.com/Kranf99/kibella/files/4636239/line_error.zip)

**Comment by Kranf99** [comment]
Actually, the error that needs to be fixed is not even in the "line" chart.
It's already inside the bar chart.

With the default parameters, the bar chart should look like this:
![2020-05-15_200826](https://user-images.githubusercontent.com/20713234/82082386-37256c80-96e8-11ea-8f27-8c399feaaf27.png)
...but, instead, it looks like this:
![2020-05-15_200748](https://user-images.githubusercontent.com/20713234/82082413-40163e00-96e8-11ea-8d55-51b490c5d041.png)
Indeed, the order is by "count", as selected here:
![aaa](https://user-images.githubusercontent.com/20713234/82082620-a26f3e80-96e8-11ea-945e-f0e5b8b03576.png)


We can change the order by "average of age", it works as expected:
![2020-05-15_201448](https://user-images.githubusercontent.com/20713234/82082698-c468c100-96e8-11ea-9275-f65d3ad92415.png)

So, to summarize, this is just the option "order by count" that does not work properly.

Also, note that this example is a little bit strange because, usually, to get this kind of chart, i would simply select a "histogram" chart and not a "group by" chart. Hopefully, the "histogram" chart works without any difficulties:
![2020-05-15_201624](https://user-images.githubusercontent.com/20713234/82082932-2b867580-96e9-11ea-9412-9430da3148fb.png)

**Comment by Kranf99** [comment]
fixed with release v3.3
